src/app/features/shared/components/molecules/navigation/DefaultNavigation.tsx

```diff
@@ -45,6 +45,9 @@ const DefaultNavigation: React.FC<Props> = ({ showSearchButton }) => {
       <ResponsiveMenuInline>
         <MenuItems />
       </ResponsiveMenuInline>
+      <ButtonLink to={to("/hulp")}>
+        <SearchButton size={50} variant="blank" iconSize={20} icon={<Handicap />} />
```

Comment:
Maybe we should replace "SearchButton" with a more generic name, something like "IconButton"

Reply:
Yes, will do